deel-2-flexbox-layout/opdracht-3/styles/styles.css

```diff
@@ -1 +1,43 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+/*Schrijf hier jouw CSS*/
+
+body {
+    display: flex;
+    height: 100vh;
+}
+
+main {
+    display: flex;
+    flex-basis: 900px;
+    justify-content: space-between;
+}
+
+.col-1 {
+    display: flex;
+    flex-basis: 500px;
+}
+
+.col-2 {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 2000px;
+}
+
+header {
+    display: flex;
```

Comment:
regel 26: is niet meer nodig, header staat in de col-2 en is daardoor automatisch een flex item geworden